tasks/transform_conformance/tests/babel-plugin-transform-react-jsx/test/fixtures/static-children/input.jsx

```diff
@@ -0,0 +1 @@
+<div><div></div></div>
```

Comment:
Can you add some comments here? Otherwise this is a really plain confusing test.

Reply:
Added